src/frontend/src/test-e2e/views.ts

```diff
@@ -168,13 +168,17 @@ export class PinAuthView extends View {
       await input.setValue(digit);
     }
   }
+
+  async waitForError(): Promise<void> {
+    await this.browser
+      .$('[data-haserror="true"]')
```

Comment:
Is this `[data-haserror="true"]` something global in II?

Could you make create a class variable `WelcomeView.ErrorSelector` and use it here?

Reply:
No, it is only used in this instance here. But I can extract a variable nonetheless. 👍